benchmark.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const path = require('path');
+const Benchmark = require('benchmark');
+const makeDir = require('make-dir');
+const tempy = require('tempy');
+const del = require('.');
+
+const suite = new Benchmark.Suite('concurrency');
+
+const tempDir = tempy.directory();
+
+const fixtures = Array.from({length: 2000}, (x, index) => {
+	return path.resolve(tempDir, (index + 1).toString());
+});
+
+function createFixtures() {
+	for (const fixture of fixtures) {
+		makeDir.sync(path.resolve(tempDir, fixture));
+	}
+}
+
+const concurrencies = [1, 3, 5, 10, 15, 20, 50, 100, Infinity];
+
+for (const concurrency of concurrencies) {
+	const name = `concurrency: ${concurrency.toString()}`;
+
+	suite.add({
+		name,
+		defer: true,
+		fn(deferred) {
+			// Can't use setup because it isn't called after every defer
+			// https://github.com/bestiejs/benchmark.js/issues/136
+			createFixtures();
+
+			// Async await was giving too many errors. stick with standard promises
+			del(['**/*'], {
+				cwd: tempDir,
+				concurrency
+				// eslint-disable-next-line promise/prefer-await-to-then
+			}).then(removedFiles => {
+				if (removedFiles.length !== fixtures.length) {
+					const error = new Error(
+						`"${name}": files removed: ${removedFiles.length}, expected: ${fixtures.length}`,
+					);
+
+					console.error(error);
+
+					del.sync(tempDir, {cwd: tempDir, force: true});
+
+					// eslint-disable-next-line unicorn/no-process-exit
+					process.exit(1);
+				}
+
+				deferred.resolve();
+			});
+		}
+	});
+}
+
+suite
+	.on('cycle', event => {
+		console.log(String(event.target));
```

Comment:
console.log already stringifies.

```suggestion
		console.log(event.target);
```

Reply:
I copied and pasted that code from an example `benchmark.js` had on their website. I just tried changing it and the console is spammed with:

```js
Benchmark {
  options: {
    async: false,
    defer: true,
    delay: 0.005,
    id: undefined,
    initCount: 1,
    maxTime: 5,
    minSamples: 5,
    minTime: 0.05,
    name: 'concurrency: 1',
    onAbort: undefined,
    onComplete: undefined,
    onCycle: undefined,
    onError: undefined,
    onReset: undefined,
    onStart: undefined,
    fn: [Function: fn]
  },
  async: false,
  defer: true,
  delay: 0.005,
  initCount: 1,
  maxTime: 5,
  minSamples: 5,
  minTime: 0.05,
  name: 'concurrency: 1',
  fn: [Function: fn],
  id: 1,
  stats: {
    moe: 0.01129889530845934,
    rme: 3.3082535827432205,
    sem: 0.005377865449052517,
    deviation: 0.02344157202437891,
    mean: 0.34153655473684214,
    sample: [
      0.397719688, 0.388559305,
      0.379714549, 0.343945989,
      0.341582504, 0.333736164,
       0.35213655, 0.321157255,
      0.327945198,  0.34182276,
      0.320502646, 0.319912421,
      0.313889747, 0.332924082,
      0.334101052,   0.3319376,
       0.33097862, 0.349683892,
      0.326944518
    ],
    variance: 0.0005495072989741439
  },
  times: {
    cycle: 0.34153655473684214,
    elapsed: 6.796,
    period: 0.34153655473684214,
    timeStamp: 1566434288783
  },
  _timerId: Timeout {
    _idleTimeout: 5,
    _idlePrev: null,
    _idleNext: null,
    _idleStart: 96,
    _onTimeout: [Function],
    _timerArgs: undefined,
    _repeat: null,
    _destroyed: false,
    [Symbol(refed)]: null,
    [Symbol(asyncId)]: 2,
    [Symbol(triggerId)]: 1
  },
  events: { complete: [] },
  running: false,
  count: 1,
  compiled: [Function: anonymous],
  cycles: 1,
  hz: 2.927944274575562
}
```